pkg/aws/client.go

```diff
@@ -63,3 +65,9 @@ func NewClientFromEC2(ec2 EC2Client) *Client {
 		ec2: ec2,
 	}
 }
+
+func NewClientFromSnowball(snowballdevice SnowballDeviceClient) *Client {
```

Comment:
What's the purpose of this constructor? it seems to build an incomplete `Client`

Reply:
for unit tests